streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/security/impl/LdapService.java

```diff
@@ -104,18 +104,18 @@ public String ldapLogin(String userId, String userPwd) {
             new InitialDirContext(ldapEnv);
           } catch (Exception e) {
             log.warn("invalid ldap credentials or ldap search error", e);
-            return null;
+            return false;
           }
           Attribute attr = attrs.next();
-          if (attr.getID().equals(ldapEmailAttribute)) {
-            return (String) attr.get();
+          if (attr.getID().equals(ldapUserIdentifyingAttribute)) {
```

Comment:
the code can be optimized  `return attr.getID().equals(ldapUserIdentifyingAttribute)`